server/src/main/java/io/crate/planner/node/ddl/AlterTableAddColumnPlan.java

```diff
@@ -78,25 +90,91 @@ public void executeOrFail(DependencyCarrier dependencies,
                               RowConsumer consumer,
                               Row params,
                               SubQueryResults subQueryResults) throws Exception {
-        BoundAddColumn stmt = bind(
-            alterTable,
-            plannerContext.transactionContext(),
-            dependencies.nodeContext(),
-            params,
-            subQueryResults,
-            dependencies.fulltextAnalyzerResolver());
+        if (plannerContext.clusterState().getNodes().getMinNodeVersion().onOrAfter(Version.V_5_1_0)) {
```

Comment:
out of curiosity, is this required? To my limited understanding, each calls become independent cluster state change requests which could be fine without it.

Reply:
I think without version we would fire new `AddColumnRequest` from some 5.1.+ (coordinator) node to a master node which executes TransportAction - and master can be <5.1 and would not be able to handle new request.